src/backend/CassandraBackend.cpp

```diff
@@ -1164,7 +1206,7 @@ CassandraBackend::open(bool readOnly)
     }
 
     auto rf = config_.valueOr<int>("replication_factor", 3);
-    auto tablePrefix = config_.valueOr<std::string>("table_prefix", "");
+    auto const tablePrefix = this->tablePrefix();
```

Comment:
minor: I don't mind adding this member function but since we don't use it other than here - consider bringing back the old code - less code for me to port to the new backend later on.